src/merge_bed.py

```diff
@@ -29,12 +26,29 @@ def read_bed_file(f: TextIO) -> list[BedLine]:
     return res
 
 
-def merge(f1: list[BedLine], f2: list[BedLine], outfile: TextIO) -> None:
+def merge(f1, f2):
     """Merge features and write them to outfile."""
-    # FIXME: I have work to do here!
-
+    if f1 == '' or f1 == None:
+        f1 = []
+    if f2 == '' or f2 == None:
+        f2 = []
```

Comment:
If we expect the input to be two lists, why are you checking for empty strings or `None`? That should never happen (and if it does, someone somewhere made a mistake, and you need to find out who did it and punish that Anton).